packages/bus-query-operation/lib/ActorQueryOperation.ts

```diff
@@ -237,7 +237,7 @@ export function getMetadata(actionOutput: IActorQueryOperationOutputStream): Pro
 }
 
 /**
- * Query operation output for a bindings stream.
+ * Query operation output for a bindings stream. (without bindings operator)
```

Comment:
The comment changes in this file are not really needed, I think they make the description too complex.